docs/reference/glossary.md

```diff
@@ -264,14 +285,23 @@ containers.
 
 ## Toolbox
 
-Docker Toolbox is the installer for Mac and Windows users.
+[Docker Toolbox](https://docs.docker.com/toolbox/overview/) is a legacy
+installer for Mac and Windows users. It uses Oracle VirtualBox for
+virtualization.
+
+For Macs running OS X El Capitan 10.11 and newer macOS releases, [Docker for
+Mac](https://docs.docker.com/docker-for-mac/) is the better solution.
+
+For Windows 10 systems that support Microsoft Hyper-V (Professional, Enterprise
```

Comment:
@mstanleyjones I revised this per @friism comments, should be accurate and address your comments too